tracing-attributes/src/lib.rs

```diff
@@ -75,8 +75,12 @@ use syn::{
 /// Instruments a function to create and enter a `tracing` [span] every time
 /// the function is called.
 ///
-/// The generated span's name will be the name of the function, and any
-/// arguments to that function will be recorded as fields using `fmt::Debug`.
+/// The generated span's name will be the name of the function. Any arguments
+/// to that function will be recorded as fields using `fmt::Debug`. To skip
+/// recording a function's or method's argument, pass the argument's name
+/// to the `skip` argument on the `#[instrument]` macro. Note that:
+/// - multiple argument names can be passed to `skip`.
+/// - arguments passed to `skip` do _not_ need to implement `fmt::Debug`.
```

Comment:
Nit/TIOLI: how about something like
> For example, `skip` can be used when an argument to an instrumented function does not implement `fmt::Debug`, or to exclude an argument with a verbose or costly `Debug` implementation.

right after you introduce  `skip`?

Reply:
good idea.